frontend/src/app/services/canvas.service.ts

```diff
@@ -105,6 +105,15 @@ export class CanvasService {
     }
   }
 
+  async deleteComment(commentID: string, postID: string){
```

Comment:
Shouldn't we be sending out the `COMMENT_REMOVE` event through socket service here?

Reply:
Sorry I'm not exactly sure how the socket service works yet. What would the COMMENT_REMOVE event do?

Reply:
It would essentially alert all users connected to the board that a comment was deleted, so they can update the comment count for the respective post on the canvas. You just have to add something like this `socketService.emit(SocketEvent.COMMENT_REMOVE, comment);`